package.json

```diff
@@ -36,5 +36,8 @@
 		"lint-staged": "13.0.4",
 		"prettier": "2.7.1",
 		"typescript": "4.8.4"
+	},
+	"dependencies": {
+		"@tupilabs/vue-lumino": "^1.1.4"
```

Comment:
This looks weird, shouldn't the library be added in the hmi-client's package.json?

Reply:
Yes those should be added inside hmi-client folder

Reply:
ah I was in the wrong dir. Got rid of it here btw